### PR TITLE
add FF admonition for Azure

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Adds our feature flag admonition to pages for Azure for BYOC & Dedicated + Azure Private Link
Based on [this thread](https://redpandadata.slack.com/archives/C0191NM4WU9/p1725418000149899).

Review deadline: Sept 5

## Page previews
[Create BYOC](https://deploy-preview-46--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/byoc/create-byoc-cluster-azure/)
[Create Dedicated](https://deploy-preview-46--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/dedicated/create-dedicated-cloud-cluster/)
[Configure Azure Private Link](https://deploy-preview-46--rp-cloud.netlify.app/redpanda-cloud/networking/azure-private-link/)

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by Feediver1** [comment]
@micheleRP Ehat is the deadline for reviewing this PR?

**Review by Feediver1 — APPROVED** [review]

**Comment by micheleRP** [comment]
> @micheleRP What is the deadline for reviewing this PR?

I just added Sept 5 since UX is confusing. Based on [this thread](https://redpandadata.slack.com/archives/C0191NM4WU9/p1725418000149899).

**Review by c4milo — APPROVED** [review]

File: modules/get-started/pages/cluster-types/dedicated/create-dedicated-cloud-cluster.adoc [hunk 1]
```diff
@@ -9,12 +9,18 @@ After you log in to https://cloud.redpanda.com[Redpanda Cloud^], you land on the
...
 +
 [NOTE]
 ====
 * If you plan to create a private network in your own VPC, select the region where your VPC is located.
 * Three availability zones provide two backups in case one availability zone goes down.
 ====
++
+[IMPORTANT]
+====
+To unlock this feature for Azure, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support^].
```
[the remark this answers is not in the document]
Comment (micheleRP) [reply modules/get-started/pages/cluster-types/dedicated/create-dedicated-cloud-cluster.adoc:22]: cc @gavinheavyside

[the remark this answers is not in the document]
Comment (gavinheavyside) [reply modules/get-started/pages/cluster-types/dedicated/create-dedicated-cloud-cluster.adoc:22]: Azure as a whole is feature flagged, but an organisation which has the azure feature flag enabled has access to both BYOC and dedicated, there is no additional feature flag for Azure dedicated

**Pull request merged**